src/successModal/index.tsx

```diff
@@ -1,100 +1,84 @@
 import styled from 'styled-components'
-import { color, font, space, modalSize } from '_utils/branding'
 import SuccessModal from './SuccessModal'
 
-const footerHeight = '96px' /* = padding + content */
+import { color, space, responsiveBreakpoints } from '_utils/branding'
 
-const StyledSuccessModal = styled(SuccessModal)`
-  & .kirk-successModal {
-    background-color: ${color.successBackground};
-    color: ${color.white};
-  }
+import IllustratedSection from 'layout/section/illustratedSection'
 
-  & .kirk-modal--large {
-    text-align: center;
-  }
+import Modal from 'modal'
+import Button from 'button'
+import TextDisplay1 from 'typography/display1'
 
-  & .kirk-modal.kirk-successModal .kirk-modal-dialog {
+const StyledSuccessModal = styled(Modal)`
+  padding: 0;
+  text-align: center;
+  background-color: ${color.successBackground};
+
+  .kirk-modal-dialog {
+    display: flex;
+    justify-content: center;
+    padding: 0;
     margin: 0 auto;
-    max-width: ${modalSize.m};
-    height: 100%;
-    padding: ${space.xl} 0 ${footerHeight} 0;
+    width: 100%;
+    border-radius: 0;
     background-color: ${color.successBackground};
   }
 
-  /* Stronger selector to override Modal component styling */
-  &.kirk-modal-dimmer .kirk-modal.kirk-successModal .kirk-modal-dialog.kirk-modal-dialog {
-    width: auto;
-  }
-
-  & .kirk-modal--large.kirk-successModal .kirk-modal-dialog {
-    padding: ${space.xl} 0;
-  }
-
-  & .kirk-modal.kirk-successModal .kirk-modal-body {
-    padding: ${space.xl};
-    font-size: ${font.l.size};
-    line-height: ${font.l.lineHeight};
-    text-align: center;
-    align-items: center;
-  }
-
-  & .kirk-modal--large.kirk-successModal .kirk-modal-body {
-    position: relative;
+  .kirk-modal-body {
     display: flex;
-  }
-
-  & .kirk-modal--large .kirk-successModal-bodyItem {
-    width: 50%;
-  }
+    min-height: 100vh;
+    flex-direction: column;
 
-  & .kirk-modal--large .kirk-successModal-bodyItem:nth-child(odd) {
-    padding-right: ${space.xl};
-  }
-
-  & .kirk-modal--large .kirk-successModal-bodyItem:nth-child(even) {
-    padding-left: ${space.xl};
-  }
-
-  & .kirk-successModal-image {
-    width: 80%;
-    margin-bottom: ${space.xl};
+    @media (${responsiveBreakpoints.isMediaLarge}) {
+      flex-direction: row;
+      align-items: center;
+      flex-wrap: wrap;
+    }
   }
+`
 
-  & .kirk-modal--small .kirk-successModal-image {
-    max-height: 33vh;
-    object-fit: contain;
-  }
+const StyledIllustratedSection = styled(IllustratedSection)`
```

Comment:
I don't feel confortable overriding and coupling illustrated section inside the SuccessModal styles.
Components should be as orthogonal as possible and here we are introducing a very deep coupling between their CSS (which is the worst coupling to maintain and test)

Reply:
Ideally, the modal should just be a particular case of a kairos page e.g.
<MyModal>
  <TheVoice>
  <IllustratedSection />
  <ButtonSection />
</MyModal>

We could have some generic CSS for all modal to manage the layout but not a bunch of CSS that couple a modal type to components inside (like you have between the succcessmodal and illustrated section).

Once more, I did not touch the layout part for modals and modal components. But the fact that we write a bunch more of styles (either it's CSS or Styled component) for only one case is a sign that's it's not generic enough and maybe not a good direction.

Reply:
🤷‍♀️ Well `IllustratedSection` does not support centred layouts (And that is why I've been proposing grid components pour manage layout instead of overriding components layout style with other styles).

You can check my other draft PR to see the solution without `IllustratedSection`. I've only applied what we have discussed in chapter to avoid creating technical debt by creating the same component twice.